docs/docsite/rst/development_guide.rst

```diff
@@ -128,3 +128,34 @@ directory structure like this:
 For further details regarding the ansible collection path configuration see the
 documentation. (`Ansible Collection Path Doc
 <https://docs.ansible.com/ansible/latest/reference_appendices/config.html#collections-paths>`__)
+
+
+Reviewing Code
+=================
+
+Prerequisite for a successful review is to have setup your environment according
+to the section *Local Setup*. To review changes of other contributors use these
+steps as a review guideline:
+
+1. Check out the new branch
```

Comment:
Wouldn't the reviewer of the PR have to first check out the forked repository of the person that created the PR.
If that is the case is there any way we can simplify this?

Or is the workflow really:
- clone the forked repo
- switch to the desired branch
- run tests/ build-doc, etc?

Reply:
Thanks for the hint. I added the missing fork checkout to the first step.